### ISSUE TITLE
Replace Non-508 Compliant PDF Documentation in DAWSON

### ISSUE THREAD

**Issue by cholly75** [ustaxcourt/ef-cms#8160]
As a user with physical or mental impairment, so that I can use DAWSON appropriately, I need documentation in DAWSON to be section 508-compliant.

There is one instance of documentation in the DAWSON application that are not currently Section 508 compliant and cannot be effectively used by accessibility tools such as screen reader/narrators. The USTC should conform to Sec. 508 compliance rules as a government agency.

The Court will provide updated versions of the documentation to replace the currently existing ones in the app.

## Acceptance Criteria
* "Searching for Documents in DAWSON" is replaced with a compliant version of the documentation

## Notes
* Updated document: [https://app.zenhub.com/files/152320868/afe06bcd-9132-4b1d-889a-479b3aec151a/download](https://app.zenhub.com/files/152320868/afe06bcd-9132-4b1d-889a-479b3aec151a/download)

## Test Cases
* "Searching for Documents in DAWSON" document has been replaced with new version in DAWSON

## Story Definition of Ready (Created on 9/26/21)
The following criteria must be met in order for the user story to be picked up by the Flexion development team.
The user story must: 
- [ ]  Be immediately actionable for the development team
- [ ]  Be focused on users
- [ ]  Include a narrative. “As a ____, so that I can ______, I need _____.”
- [ ]  Include user-centric acceptance criteria that expresses the users' needs
- [ ]  Include user-centric testing scenarios
- [ ]  Include preconditions (story dependencies)
- [ ]  Not include design or technical solutions
- [ ]  Include non-standard testing, deployment, documentation, browser, mobile or security requirements, as well as any requirements for 3rd-party integrations

Process: Flexion developers and designers will test if the story meets acceptance criteria and test cases in Flexion dev and staging environments (“standard testing”). If additional acceptance criteria or testing scenarios are discovered while the story is in progress, a new story should be created, added to the backlog and prioritized by the product owner. 

## Definition of Done (Updated 10-6-21)
**Product Owner**
 - [ ]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios have been refined to meet all acceptance criteria 
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [ ] Automated test scripts have been written
 - [ ] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [ ] Verify that language for docket record for internal users and external users is identical
 - [ ] New screens have been added to pa11y scripts
 - [ ] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [ ] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [ ] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [ ] Interactors should validate entities before calling persistence methods
 - [ ] Code refactored for clarity and to remove any known technical debt
 - [ ] Acceptance criteria for the story has been met
 - [ ] If there are special instructions in order to deploy into the next environment, add them as a comment in the story
 - [ ] Deployed to any Flexion environment for Engineering Review
 - [ ] Deployed to the Court's migration environment

**Comment by ttlenard** [comment]
## Test Cases

#### 1) "Searching for Documents in DAWSON" document has been replaced with new version in DAWSON
- Go to the DAWSON public user dashboard
- Click on the Order tab
- Click on the "Learn more about searching in DAWSON" URL in the "How to Search" tile
- Repeat these same steps by clicking on the Opinion tab

Expected Results
- [ ] The updated 508 compliant document opens in a new tab

**Comment by swongCO** [comment]
UX doesn't have the proper tools to confirm the pdf is 508 compliant but was able to confirm that the pdf opened up in new tab for both Order and Opinion.
